components/progresscircle/stories/progresscircle.stories.js

```diff
@@ -102,3 +118,25 @@ StaticWhiteIndeterminate.args = {
 StaticWhiteIndeterminate.parameters = {
 	chromatic: { disableSnapshot: true },
 };
+
+export const StaticBlackDeterminate = Template.bind({});
+StaticBlackDeterminate.tags = ["!dev"];
+StaticBlackDeterminate.storyName = "Static black, default";
+StaticBlackDeterminate.args = {
+	staticColor: "black",
+	isIndeterminate: false,
+};
+StaticBlackDeterminate.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const StaticBlackIndeterminate = Template.bind({});
+StaticBlackIndeterminate.tags = ["!dev"];
+StaticBlackIndeterminate.storyName = "Static black, indeterminate";
+StaticBlackIndeterminate.args = {
+	staticColor: "black",
+	isIndeterminate: true,
+};
+StaticBlackIndeterminate.parameters = {
+	chromatic: { disableSnapshot: true },
+};
```

Comment:
Now that we do have the `Sizing()` function, what do you tihnk about using it for both of these static black stories? Instead of binding to `Template()`, bind to `Sizing()` like the static white stories? 

```suggestion
export const StaticBlackDeterminate = Sizing.bind({});
StaticBlackDeterminate.tags = ["!dev"];
StaticBlackDeterminate.storyName = "Static black, default";
StaticBlackDeterminate.args = {
	staticColor: "black",
	isIndeterminate: false,
};
StaticBlackDeterminate.parameters = {
	chromatic: { disableSnapshot: true },
};

export const StaticBlackIndeterminate = Sizing.bind({});
StaticBlackIndeterminate.tags = ["!dev"];
StaticBlackIndeterminate.storyName = "Static black, indeterminate";
StaticBlackIndeterminate.args = {
	staticColor: "black",
	isIndeterminate: true,
};
StaticBlackIndeterminate.parameters = {
	chromatic: { disableSnapshot: true },
};
```

There's nothing wrong with what you have, it's just the odd ones out of the bunch on the docs page.

<img width="1058" alt="Screenshot 2025-01-08 at 4 17 36 PM" src="https://github.com/user-attachments/assets/743b2943-72e6-4b8b-924e-d8ce3c97e638" />